lib/scholar/decomposition/pca.ex

```diff
@@ -471,7 +471,8 @@ defmodule Scholar.Decomposition.PCA do
     end
   end
 
-  defnp flip_svd(u, v) do
+  @doc false
+  defn flip_svd(u, v) do
```

Comment:
I would pull this out in a separate module in the same folder, e.g. `Scholar.Decomposition.Utils` or `Scholar.Decomposition.Shared`. See how it's done in `Scholar.Neighbors.Utils`.